### ISSUE TITLE
Error in building : setupDB FAILED

### ISSUE THREAD

**Issue by codekaust** [draeger-lab/ModelPolisher#36]
I am using Linux Ubuntu 18.04, Java version "1.8.0_211", Gradle 5.0
While tring to build ModelPolisher, I run command `gradle` with current directory being `ModelPolisher/`, build fails with the following output:
```
Starting a Gradle Daemon, 1 stopped Daemon could not be reused, use --status for details

> Configure project :
Setup DB

> Task :compileJava
Note: /home/kaustubh/Desktop/ModelPolisher/src/edu/ucsd/sbrg/util/SBMLUtils.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /home/kaustubh/Desktop/ModelPolisher/src/edu/ucsd/sbrg/cobra/COBRAparser.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.

> Task :setupDB FAILED
scripts/configureSQLiteDB.sh: 9: scripts/configureSQLiteDB.sh: Bad substitution
scripts/configureSQLiteDB.sh: 24: scripts/configureSQLiteDB.sh: Syntax error: "(" unexpected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':setupDB'.
> Process 'command 'sh'' finished with non-zero exit value 2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 6s
3 actionable tasks: 2 executed, 1 up-to-date
```

Also, note running `sh ./scripts/configureSQLiteDB.sh` shows the following error:
```
./scripts/configureSQLiteDB.sh: 9: ./scripts/configureSQLiteDB.sh: Bad substitution
./scripts/configureSQLiteDB.sh: 24: ./scripts/configureSQLiteDB.sh: Syntax error: "(" unexpected

```

**Comment by matthiaskoenig** [comment]
Provide more information.
Especially which operation system and which gradle version? 

I tried this on ubuntu linux last week and had no issues. So probably related to Windows or Mac. You have to make sure that you are not breaking the other OS support by your fixes!

**Comment by codekaust** [comment]
Thank You @matthiaskoenig . I have updated the issue description.

**Comment by codekaust** [comment]
Resolved in 66d5ff27f3a38e1a9bb02a9fc85e6ae364836437.
